Gulpfile.js

```diff
@@ -1,28 +1,17 @@
-var _ = require('underscore');
-var assets  = require('postcss-assets');
-var autoprefixer = require('autoprefixer-core');
-var deploy = require('gulp-gh-pages');
-var es = require('event-stream');
 var gulp = require('gulp');
-var karma = require('gulp-karma');
-var livereload = require('gulp-livereload');
-var ngAnnotate = require('gulp-ng-annotate');
-var postcss = require('gulp-postcss');
-var protractor = require('gulp-protractor').protractor;
-var rjs = require('gulp-requirejs');
-var sass = require('gulp-sass');
-var spawn = require('child_process').spawn;
-var uglify = require('gulp-uglify');
+var plumber = require('gulp-plumber');
 var webdriver = require('gulp-protractor').webdriver_standalone;
```

Comment:
I'm not sure about the change of require calls placements. It's the usual style for node.js to have all external modules pulled on top of the file, e.g. gulp https://github.com/gulpjs/gulp/blob/master/bin/gulp.js